### ISSUE TITLE
jrnl does not support the %z strftime directive

### ISSUE THREAD

**Issue by danielhoherd** [maebert/jrnl#444]
jrnl does not appear to support the %z (timezone offset) strftime directive in its time format.

```
$ grep time ~/.jrnl_config
  "timeformat": "%F %T%z",
$ jrnl testing
[Entry added to default journal]
$ cat journal.txt
2016-10-21 15:03:47 testing
$ date "+%F %T%z"
2016-10-21 15:04:03-0700
```

**Comment by danielhoherd** [comment]
I just rediscovered this bug when using `jrnl`, then discovered that I had already filed it! I have verified on macOS and Linux that this is still a problem in jrnl version jrnl==1.9.8 which is the latest available through pip.

**Comment by dftchfmstrpnk** [comment]
+1

**Comment by dftchfmstrpnk** [comment]
Did some reading. Python doesn't support timezones in the stdlib.
https://stackoverflow.com/questions/4530069/python-how-to-get-a-value-of-datetime-today-that-is-timezone-aware

jrnl doesn't do this but with the above article for saving the entries and this article for displaying them, this should be pretty easy:
https://stackoverflow.com/questions/13218506/how-to-get-system-timezone-setting-and-pass-it-to-pytz-timezone

**Comment by artgromov** [comment]
for me this works just fine:

```
from datetime import datetime
TIMEZONE_LOCAL = datetime.now().astimezone().tzinfo
...
timestamp = datetime.now()
if timestamp.tzinfo is None:
    timestamp = timestamp.replace(tzinfo=TIMEZONE_LOCAL)
```

**Comment by wren** [comment]
According to @micahellison (in #630), this was a limitation of Python until 3.2. Now that we're moving on from Python 2, we'll be able to add proper `%z` support soon. We have some date reworking coming down the line, so this will probably be part of that.

**Comment by dftchfmstrpnk** [comment]
@wren What's the timeline for the time changes :)

**Comment by wren** [comment]
@dftchfmstrpnk This issue isn't scheduled in a milestone yet, but PRs are welcome if anyone wants to try to get this into a sooner release.

**Comment by artgromov** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

SRSLY??

**Comment by wren** [comment]
@artgromov Hi! You seem to care about this feature quite a bit. I understand that it can be frustrating to see something you care about not get the attention you'd like. We try to prioritize the features that users care about, but not everything can make it in all at once (remember that we only have so much time, and this is unpaid work).

So, as I mentioned above, this feature isn't currently scheduled. If you would like to see it in the project sooner than when we have time to work on it, please send us a PR. PRs are the best and fastest way to get a feature you care about into this project. Please be sure to reference this issue number in your PR so the reviewer has more context about the work.

That all being said, I would like to address your comment above. "SRSLY??" is an unprofessional and disrespectful response. I invite you to review our [code of conduct](https://github.com/jrnl-org/jrnl/blob/master/CODE_OF_CONDUCT.md). Please use this as a guide for your future comments in this project.

**Comment by artgromov** [comment]
Sorry I didn't mean to seem disrespectful at all.

I was frustrated because of "drop the issue when it becomes stale" policy.
This policy severely decreases the project's chances to improve in the future.

**Comment by peter-schmidbauer** [comment]
> This policy severely decreases the project's chances to improve in the future.
No, it makes sure only issues people care about stay open, so that maintainers and other contributors don't waste their time implementing stuff that might be not needed anymore. 
It helps the project focus on what's important.
